libc-test/build.rs

```diff
@@ -309,6 +316,11 @@ fn main() {
             // This is actually a union, not a struct
             "sigval" => true,
 
+            // These structs have changed since uClibc was developed
+            "stat" | "stat64" | "dqblk" if uclibc => true,
+            // These structs didn't exist when uClibc was developed
+            "aiocb" if uclibc => true,
```

Comment:
Would you have any advice on how to remove `aiocb`? Just sticking a `#cfg[]` above it breaks because of `s!` macro wrapping it